src/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices/RemoteAppServerOptions.cs

```diff
@@ -11,6 +10,32 @@ namespace Microsoft.AspNetCore.SystemWebAdapters;
 /// </summary>
 public class RemoteAppServerOptions
 {
+    // Matches an empty GUID (32 0s) with dashes, parentheses, and braces optional
+    private const string EmptyGuidRegex =
```

Comment:
What about creating an attribute that would extend the `RegularExpressionAttribute` and could be in the shared services? Then you wouldn't have this duplication

Reply:
That makes sense. I'll move this there.